tests/integration_tests/test_registry_spec.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+"""Integration tests for registry spec helper functions."""
+
+from __future__ import annotations
+
+import pytest
+
+from airbyte._util.registry_spec import (
+    get_connector_spec_from_registry,
+    validate_connector_config_from_registry,
+)
+
+
+@pytest.mark.parametrize(
+    "connector_name,platform",
+    [
+        ("source-faker", "oss"),
+        ("source-faker", "cloud"),
+        ("destination-duckdb", "oss"),
+    ],
+)
+def test_get_connector_spec_from_registry(connector_name: str, platform: str) -> None:
```

Comment:
Add some instances here that leverage an optional version (instead of defaulting to latest). If needed, add the necessary logic upstream to make version an optional input.